### DISCUSSION TITLE
On-screen notifications when events fire

### DISCUSSION THREAD

**Discussion by kg4zow** [Taitava/obsidian-shellcommands#417] [Q&A] [answered]
I use the "obsidian-git" plugin to sync my vaults against a git repo. It has settings which, when enabled, makes a small box appear at the top right corner of the app whenever it "does something", i.e. commit, push, fetch, etc. and then disappear a few seconds later.

It would be nice to have an option which enables this kind of notification - maybe as a *global* "on/off" thing, but ideally as something which could be enabled or disabled for each script, or for each event type *on* a script. As it is now, the script I'm running creates its own log file, and I have to `tail -F` this file to be sure it's firing when I expect it to. (And so far it has been.)

It would also be nice, although maybe a little embarrassing, to find that this functionality is already in there, and I've just missed it so far.

**Comment by kg4zow** [comment, marked as answer]
I just discovered by accident, anything that the script sends to STDERR ends up becoming one of the small black boxes I'm talking about.

**Reply by kg4zow** [reply]
And *then* I found the [documentation](https://publish.obsidian.md/shellcommands/Output+handling/Output+channels) about how the STDOUT/STDERR stuff works.

**Reply by Taitava** [reply]
Yea, and there's also [an option to show a notification box when a shell command starts to execute](https://publish.obsidian.md/shellcommands/Basic+usage/Monitoring+execution). 😉

(I'll move this discussion to _Q&A_.)

**Reply by Taitava** [reply]
To improve things a bit, I'm thinking about changing the default values for a couple of settings. I'd like to ask you @kg4zow to answer these two polls I just created, please: 🙂 
- #422
- #423
